src/route-decorators.ts

```diff
@@ -0,0 +1,39 @@
+export interface RouteXxlConfig {
```

Comment:
I think the `import 'rxjs/add/operator/map';` needs to be added. I noticed in my [stackblitz](https://stackblitz.com/edit/angular-1znv1y?file=app%2Futils%2Froute-decorators.ts) that you will get an error otherwise